spec/puppetserver/ca/action/import_spec.rb

```diff
@@ -73,7 +73,8 @@
       Dir.mktmpdir do |tmpdir|
         exit_code = subject.run({ 'cert-bundle' => File.join(tmpdir, 'cert_bundle.pem'),
                                   'private-key' => File.join(tmpdir, 'private_key.pem'),
-                                  'crl-chain' => File.join(tmpdir, 'crl_chain.pem') })
+                                  'crl-chain' => File.join(tmpdir, 'crl_chain.pem'),
+                                  'certname' => '' })
```

Comment:
What's the purpose of adding certname here?  Just to show it accepts the option and doesn't error?

Reply:
You have to. The `run` function expects that `parse` will hand it a complete hash of arguments, with all the keys present with default values. We could do a bunch of null checking in `run`, but since the library we're using for options paring already prepares the full hash, it seems unnecessary just to make testing cleaner.